scripts/release/release.sh

```diff
@@ -146,7 +146,7 @@ function apply_cherry_picks() {
 function find_last_release() {
   local branch="${1:-HEAD}"
   local baseline="${2:-$(get_release_baseline "${branch}")}"
-  local changes="$(git log --pretty=format:%H "${baseline}~".."${branch}")"
+  local changes="$(git log --pretty=format:%H "${baseline}~..${branch}")"
```

Comment:
Why that change?

Reply:
Because the inner quotes are no-ops. See [shellcheck/SC2140](https://github.com/koalaman/shellcheck/wiki/SC2140) for reference.

Reply:
On the other hand, they are nested once more.
Shouldn't it just be this?
`local changes="$(git log --pretty=format:%H ${baseline}~..${branch})"`

Reply:
just leave it how the original proposal was

Reply:
Could it be that the error was already present in the base commit?

Reply:
the inner quote are not inner quote. We end the quoting then add .. then start it again.